src/App.js

```diff
@@ -9,7 +9,7 @@ import store from './redux/Store';
 
 function App() {
   return (
-    <div className="App">
+    <div className="bg-[#f5f5f4]">
       <Provider store={store}>
         <Router>
           <Navigation />
```

Comment:
You have done really well with the styling of the app, I recommend you adjust the size of the button to make it match the design on Zeplin. I believe if this is addressed the app would look much better.

YOUR DESIGN
<img width="841" alt="Screenshot 2023-07-19 214316" src="https://github.com/AnsarIbrahim/bookstore/assets/84930355/cd5f250d-10ed-480e-b5e7-e413768925fe">

ZEPLIN TEMPLATE
<img width="385" alt="Screenshot 2023-07-19 213431" src="https://github.com/AnsarIbrahim/bookstore/assets/84930355/0b54e4a7-16aa-4bc7-bffe-aa0948bb534e">